spec/models/tag_spec.rb

```diff
@@ -3,29 +3,49 @@
 RSpec.describe Tag, type: :model do
   let(:tag) { build(:tag) }
 
-  it "passes validations if bg_color_hex is valid" do
```

Comment:
While I was in here, I collapsed a lot of these validation-related specs into a single describe block for readability.